packages/graphback/src/GraphQLBackendCreator.ts

```diff
@@ -88,27 +105,73 @@ export class GraphQLBackendCreator {
   }
 
   public async createClient(): Promise<Client> {
-    const clientGenerator = new ClientGenerator(this.inputContext)
+    const clientGenerator = new ClientGenerator(this.inputContext);
 
-    return clientGenerator.generate()
+    return clientGenerator.generate();
   }
 
+  public async migrateDatabase(
+    initializationStrategy: DatabaseInitializationStrategy
+  ): Promise<void> {
+    const context = this.inputContext.filter(
+      (t: Type) =>
+        t.kind === OBJECT_TYPE_DEFINITION &&
+        t.name !== 'Query' &&
+        t.name !== 'Mutation' &&
+        t.name !== 'Subscription',
+    );
+
+    const changes = await this.getSchemaChanges(
+      this.oldSchema,
+      this.newSchema,
+    );
+
+    if (!this.oldSchema || changes.length > 0) {
+      this.schemaContext.updateOldSchema(this.newSchema);
+    }
 
-  public async createDatabase(): Promise<void> {
-    const context = this.inputContext.filter((t: Type) => t.kind === OBJECT_TYPE_DEFINITION && t.name !== 'Query' && t.name !== 'Mutation' && t.name !== 'Subscription')
     try {
       if (this.dataLayerManager) {
-        logger.info("Creating database structure")
-        await this.dataLayerManager.createDatabaseResources(this.dbContextProvider, context);
-        await this.dataLayerManager.createDatabaseRelations(this.dbContextProvider, context);
+        if (initializationStrategy === DatabaseInitializationStrategy.DropCreate || !this.oldSchema) {
+          logger.info('Creating database structure');
+          await this.dataLayerManager.createDatabaseResources(
```

Comment:
[Question] Do you think that it will be possible to repurpose them to initially create database for dev reasons but not drop the actual database when db is there?